src/module/TodoStatusSection.js

```diff
@@ -0,0 +1,100 @@
+import { STATUS, CSS_CLASS, STRING_LITERAL, EVENT } from './Constant';
+
+export default class TodoStatusSection {
+  constructor(eventEmitter) {
+    this.eventEmitter = eventEmitter;
+
+    this.createTodoStatusDomElement();
+  }
+
+  createTodoStatusDomElement() {
+    this.statusElement = document.createElement('div');
+    this.activeCounter = document.createElement('span');
+    this.createFilterElement();
+    this.createClearCompletedBtnElement();
+
+    this.activeCounter.id = 'active-counter';
+    this.activeCounter.innerText = `0 ${STRING_LITERAL.ITEMS_LEFT}`;
+
+    this.statusElement.className = CSS_CLASS.TODO_STATUS;
+    this.statusElement.append(this.activeCounter, this.filterDiv, this.clearCompletedBtn);
+  }
+
+  createFilterElement() {
+    this.filterDiv = document.createElement('div');
+    this.filterAll = document.createElement('button');
+    this.filterActive = document.createElement('button');
+    this.filterCompleted = document.createElement('button');
+
+    this.filterDiv.className = CSS_CLASS.FILTER;
+
+    this.filterAll.id = 'filter-all';
+    this.filterAll.className = STATUS.SELECTED;
+    this.filterAll.innerText = 'All';
+    this.filterAll.addEventListener('click', this.setClassNameToFilterAllElement.bind(this));
```

Comment:
```suggestion
    this.filterAll = createFilterButton({
      id,
      className,
      innerText,
      onClick
    })
```

다른 세 버튼도 공통 메서드로 분리할 수 있지 않을까요~?